Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Iterative_authalic_parameterizer_3.h

```diff
@@ -632,41 +631,33 @@ class Iterative_authalic_parameterizer_3
   /// \param mesh a triangulated surface.
   /// \param main_vertex_v_i the vertex of `mesh` with index `i`
   /// \param neighbor_vertex_v_j the vertex of `mesh` with index `j`
-  NT compute_w_ij(const Triangle_mesh& tmesh,
+  NT compute_w_ij(const Triangle_mesh& mesh,
                   vertex_descriptor main_vertex_v_i,
                   Vertex_around_target_circulator<Triangle_mesh> neighbor_vertex_v_j) const
   {
-    const PPM ppmap = get(vertex_point, tmesh);
-
-    const PPM_ref position_v_i = get(ppmap, main_vertex_v_i);
-    const PPM_ref position_v_j = get(ppmap, *neighbor_vertex_v_j);
+    const PPM ppmap = get(vertex_point, mesh);
+    const Point_3& position_v_i = get(ppmap, main_vertex_v_i);
+    const Point_3& position_v_j = get(ppmap, *neighbor_vertex_v_j);
 
-    // Compute the square norm of v_j -> v_i vector
-    Vector_3 edge = position_v_i - position_v_j;
-    NT square_len = edge*edge;
+    const Vector_3 edge = position_v_i - position_v_j;
+    const NT squared_length = edge * edge;
 
-    // Compute cotangent of (v_k,v_j,v_i) corner (i.e. cotan of v_j corner)
-    // if v_k is the vertex before v_j when circulating around v_i
     vertex_around_target_circulator previous_vertex_v_k = neighbor_vertex_v_j;
-    --previous_vertex_v_k;
-    const PPM_ref position_v_k = get(ppmap, *previous_vertex_v_k);
-//    NT cotg_psi_ij = internal::cotangent<Kernel>(position_v_k, position_v_j, position_v_i);
-    NT cotg_beta_ij = internal::cotangent<Kernel>(position_v_i, position_v_k, position_v_j);
+    previous_vertex_v_k--;
```

Comment:
Why the change from `--previous_vertex_v_k;`? This was faster than the new format.

Reply:
The same as above. I copied from other parameterizers where the other version was used. I fixed it everywhere but in my experience the speedup is negligible.